renku/core/management/workflow/plan_factory.py

```diff
@@ -395,6 +397,12 @@ def add_command_output(
 
         mapped_stream = self.get_stream_mapping_for_value(default_value)
 
+        if mapped_stream and not position:
```

Comment:
Does it matter if we check `not position` instead of `position is None`?

Reply:
it shouldn't, position always starts at 1. but `is None` is cleaner, I'll change it